packages/libs/eda/src/lib/map/analysis/littleFilters.ts

```diff
@@ -0,0 +1,50 @@
+import { Filter, StudyEntity, Variable } from '../../core';
+import { useMemo } from 'react';
+import { AppState } from './appState';
+import { GeoConfig } from '../../core/types/geoConfig';
+import { useDeepValue } from '../../core/hooks/immutability';
+import { VariableDescriptor } from '../../core/types/variable';
+
+export interface useLittleFiltersProps {
```

Comment:
Please make this start with an uppercase letter

Reply:
All done - thanks for looking